incore_auth/app.py

```diff
@@ -123,6 +125,23 @@ def update_services_thread(request_info):
             })
             app.logger.info(f"Inserted space document for {username}")
 
+        mongo_usage = mongo_client["spacedb"]["UserAllocations"].find_one({"username": username})
```

Comment:
is this what is done by the cronjob?

Reply:
maybe...But that's the thing related to this bug #38